clients/drcachesim/tracer/func_trace.cpp

```diff
@@ -125,24 +125,53 @@ func_post_hook(void *wrapcxt, void *user_data)
     append_entry(drcontext, TRACE_MARKER_TYPE_FUNC_RETVAL, retval);
 }
 
+static app_pc
+get_pc_by_symbol(const module_data_t *mod, const char *symbol)
+{
+    // Try to find the symbol in the dynamic symbol table.
+    app_pc pc = (app_pc)dr_get_proc_address(mod->handle, symbol);
+    if (pc != NULL) {
+        dr_fprintf(STDERR, "dr_get_proc_address found symbol %s at pc=%p\n", symbol, pc);
```

Comment:
Users do not like noisy messages that can't be disabled.  Use NOTIFY().  Ditto below in several places.

Reply:
Done.